services/brig/src/Brig/Provider/RPC.hs

```diff
@@ -60,10 +64,23 @@ data ServiceError
 -- or the response body cannot be parsed, a 'ServiceError' is returned.
 createBot :: ServiceConn -> NewBotRequest -> ExceptT ServiceError AppIO NewBotResponse
 createBot scon new = do
-    mg <- ($ toList (sconFingerprints scon)) <$> view extGetManager
+    let fprs = toList (sconFingerprints scon)
+    (man, verifyFingerprints) <- view extGetManager
     extHandleAll onExc $ do
-        rs <- lift $ recovering x3 httpHandlers $ const $
-            liftIO $ Http.httpLbs req mg
+        rs <- lift $ recovering x3 httpHandlers $ const $ liftIO $
+            -- TODO: not sure if 'Reuse' is the right thing to do here --
+            -- maybe instead of returning the connection to the pool we want
+            -- to close it
```

Comment:
> maybe instead of returning the connection to the pool we want to close it

In case of popular services, keeping a connection open can/should be done; I expect the manager to do the right thing here and keep connections open as needed, etc; by default, it keeps up to 512 connections open across all hosts which sounds reasonable for our use case